app/services/flow/flow_state_machine.rb

```diff
@@ -173,6 +173,14 @@ def redirect_to_step(step)
       redirect_to send(@step_url, step: step)
     end
 
+    def opt_in_analytics_properties
+      extra = {}
+      if IdentityConfig.store.in_person_proofing_opt_in_enabled
+        extra = { opted_in_to_in_person_proofing: idv_session.opted_in_to_in_person_proofing }
+      end
+      extra
```

Comment:
ditto and +1 to adding a mixin
```suggestion
      if IdentityConfig.store.in_person_proofing_opt_in_enabled
        { opted_in_to_in_person_proofing: idv_session.opted_in_to_in_person_proofing }
      else
        {}
      end
```